docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -199,6 +199,8 @@ NOTE: Adding a `%prod.` profile prefix to `quarkus.oidc.auth-server-url` ensures
 
 NOTE: By default, applications using the `quarkus-oidc` extension are marked as a `service` type application (see `quarkus.oidc.application-type`). This extension also supports only  `web-app` type applications but only if the access token returned as part of the authorization code grant response is marked as a source of roles: `quarkus.oidc.roles.source=accesstoken` (`web-app` type applications check ID token roles by default).
 
+NOTE: The configurated `quarkus-realm.json` file for this example can be found in https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-keycloak-authorization-quickstart/config/quarkus-realm.json[quarkus-quickstarts/security-keycloak-authorization-quickstart/config]
```

Comment:
```suggestion
```

I'm not sure adding this note does not introduce duplication as just a few lines below there is 

Import the [realm configuration file](https://github.com/quarkusio/quarkus/blob/6dc006f7d86d6f9129997ca6263dd79f82213c79/docs/src/main/asciidoc/%7Bquickstarts-tree-url%7D/security-keycloak-authorization-quickstart/config/quarkus-realm.json)

Reply:
Ok, I'll drop this note

Reply:
Dropped